lib/suspenders/generators/app_generator.rb

```diff
@@ -177,12 +178,16 @@ def remove_routes_comment_lines
       build :remove_routes_comment_lines
     end
 
+    def configure_circleci
+      say "Configuring CircleCI"
+      build :configure_circleci
+    end
+
     def generate_default
       run("spring stop")
       generate("suspenders:static")
       generate("suspenders:stylesheet_base")
       generate("suspenders:testing")
-      generate("suspenders:ci")
```

Comment:
We'll need to keep this in the generator, so people can remove CI support if they don't want it (or, for people who just add the gem to their Gemfile, so people can add CI support if they want it).